### DISCUSSION TITLE
Two ADS1115 used with ESP32 are affected.

### DISCUSSION THREAD

**Discussion by sawacs** [arendst/Tasmota#12042] [Q&A] [answered]
Hi.
Two ADS1115 used with ESP32 are affected.
The web interface displays all values ​​from ADS1115 at address 48 and all values ​​from ADS1115 at address 49.
![Screenshot_20210507_220959_com android chrome](https://user-images.githubusercontent.com/48694194/117503712-a2172580-af81-11eb-9966-42831fed3f76.jpg)

I am trying to use the read values ​​from two ADS1115 converters in my script! I am using the variables ADS1115-48#A0 (A1, A2, A3) and ADS1115-49#A0 (A1, A2, A3) and always get 0 instead of a specific value like on WWW. Where do I go wrong.
I am attaching the log from the terminal:
SENSOR = {"Time": "2021-05-07T20: 45: 42", "ANALOG": {"A1": 0, "A2": 3537}, "ENERGY": {"TotalStartTime": "2021-05 -07T06: 25: 14 "," Total ": 21478.113," Yesterday ": 0.000," Today ": 0.000," Period ": 0," Power ": 0," Current ": 0.000}," ADS1115-48 " : {"A0": 13397, "A1": 13371, "A2": 13333, "A3": 13388}, "ADS1115-49": {"A0": 13414, "A1": 13403, "A2": 13407, "A3": 13389}, "ESP32": {"Temperature": 53.3}, "TempUnit": "C"}

My test script:
[script-test.txt](https://github.com/arendst/Tasmota/files/6444108/script-test.txt)

**Comment by sfromis** [comment, marked as answer]
To make it work in a script, you may have to use [SetOption64](https://tasmota.github.io/docs/Commands/#setoption64) to change the separator to `_` - this avoids the `-` being seen as subtraction.

**Comment by sawacs** [comment]
Thank's.
It works...!

pt., 7 maj 2021 o 22:54 sfromis ***@***.***> napisał(a):
